blockchain/transaction_svc.py

```diff
@@ -113,6 +113,13 @@ def set_txn_status(self, log, status_code):
             log.error("500: Internal Error: The server encountered an unexpected condition which prevented it from fulfilling the request.")
         self.set_status(status_code)
 
+# TODO: implement handler for posting transactions to bitcoin network
```

Comment:
Just something I forgot to take out when we decided to not implement that in this ticket.